datastores-parent/datastore-tests/pom.xml

```diff
@@ -76,5 +76,10 @@
 			<artifactId>websocket-jakarta-server</artifactId>
 			<scope>compile</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-core</artifactId>
+			<scope>compile</scope>
```

Comment:
please drop this `compile` scope (default `test` will be used in this case :)

Reply:
The tests are in `datastores-parent/datastore-tests/src/main/java/...` I do not remember why it has been done this way. 
The build fails now.
IMO the tests should be moved to `src/test/java/` but reverting the scope to `compile` is OKish too for this PR!

Reply:
Exactly. It does not work with `test` scope at the moment.

Reply:
My bad :(
I didn't noticed this is separate `tests` module

the scope should be `compile`

sorry for the noise :(